app/controllers/studies_controller.rb

```diff
@@ -579,7 +579,7 @@ def parse
 
   # method to download files if study is private, will create temporary signed_url after checking user quota
   def download_private_file
-    @study = Study.find_by(accession: params[:accession], url_safe_name: params[:study_name])
+    @study = Study.find_by(accession: params[:accession])
```

Comment:
```suggestion
    @study = Study.find_by!(accession: params[:accession])
```
might not be necessary if there are other checks beforehand, but probably can't hurt

Reply:
No, good point.  There aren't other checks, so this would fail downstream.  At least this way it will fail with a predictable error or `Record Not Found` from Mongo.

Reply:
I went a different route - we handle missing studies and redirect for "public" download requests, so I'm applying similar logic here.